src/validation/stateful.rs

```diff
@@ -1,5 +1,5 @@
 use crate::model;
 
-pub fn isValid(model::Transaction) -> bool {
+pub fn isValid(tx: model::Transaction) -> bool {
```

Comment:
Should be `is_valid` according to rust naming convention.

Reply:
old code, will fix warnings in next PR